example/classifier/odp_classifier.c

```diff
@@ -784,6 +784,8 @@ int main(int argc, char *argv[])
 		ODPH_ERR("err: odp_pool_destroy error\n");
 
 args_error:
+	if (args->if_name)
+		free(args->if_name);
```

Comment:
If parse_args() fails, it already frees args->ifname, so in that case this introduces a double free.